inst/cpp_generation/model_input.csv

```diff
@@ -28,11 +28,32 @@ paed_cd4_prog,hc1_cd4_prog,real_type,ModelVariant::run_child_model,,FALSE,1,chil
 adol_cd4_prog,hc2_cd4_prog,real_type,ModelVariant::run_child_model,,FALSE,1,children.hc2DS,,,
 ctx_effect,ctx_effect,real_type,ModelVariant::run_child_model,,FALSE,1,1,,,
 ctx_val,ctx_val,real_type,ModelVariant::run_child_model,,FALSE,1,proj_years,,,
-paed_art_elig_age,hc_art_elig_age,real_type,ModelVariant::run_child_model,,FALSE,1,proj_years,,,
+paed_art_elig_age,hc_art_elig_age,int,ModelVariant::run_child_model,,FALSE,1,proj_years,,,
 paed_art_elig_cd4,hc_art_elig_cd4,real_type,ModelVariant::run_child_model,,TRUE,2,options.p_idx_hiv_first_adult,proj_years,,
 mort_art_rr,hc_art_mort_rr,real_type,ModelVariant::run_child_model,,FALSE,3,base.hTS,options.p_idx_hiv_first_adult,proj_years,
 paed_art_mort,hc1_art_mort,real_type,ModelVariant::run_child_model,,FALSE,3,children.hc1DS,base.hTS,children.hc1AG,
 adol_art_mort,hc2_art_mort,real_type,ModelVariant::run_child_model,,FALSE,3,children.hc2DS,base.hTS,children.hc2AG,
 artpaeds_isperc,hc_art_isperc,int,ModelVariant::run_child_model,,FALSE,1,proj_years,,,
-paed_art_val,hc_art_val,real_type,ModelVariant::run_child_model,,FALSE,1,proj_years,,,
+paed_art_val,hc_art_val,real_type,ModelVariant::run_child_model,,FALSE,2,4,proj_years,,
 init_art_dist,hc_art_init_dist,real_type,ModelVariant::run_child_model,,FALSE,2,options.p_idx_hiv_first_adult,proj_years,,
+adult_cd4_dist,adult_cd4_dist,real_type,ModelVariant::run_child_model,,FALSE,2,base.hDS,children.hc2DS,,
```

Comment:
re variable naming, in the README there is a section "Leapfrog to frogger glossary" which has some naming rules. We added a set of prefixes there. I think fine if this is a new prefix for `adult` and `fert`. If so can we add to the README? But if not then I think let's try to move these to use those standardised prefixes?

Reply:
That said - up to you. I don't feel super strongly about this if you think your names are easier to read and understand.